flow/workflows/cdc_flow.go

```diff
@@ -568,3 +574,15 @@ func CDCFlowWorkflow(
 		}
 	}
 }
+
+func GetFlowMetadataContext(ctx workflow.Context, flowJobName string, sourceName string, destinationName string) (workflow.Context, error) {
```

Comment:
there's another utility method around in these workflow files (GetSideEffect?), these could be in a util.go